test/snippet/range/view/range_view_to_rank.cpp

```diff
@@ -0,0 +1,28 @@
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/alphabet/quality/phred42.hpp>
+#include <seqan3/alphabet/quality/aliases.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/range/view/to_rank.hpp>
+#include <seqan3/range/view/convert.hpp>
+int main()
```

Comment:
add newline inbetween

Reply:
not changed yet

Reply:
I'm confused, it is definitely resolved on my end... Are you looking at outdated files somehow?